tests/integration/test_hedged_requests_parallel/test.py

```diff
@@ -218,7 +218,7 @@ def test_combination2(started_cluster):
         node_4_sleep_in_send_tables_status=1000,
     )
     check_query()
-    check_changing_replica_events(4)
+    check_changing_replica_events(3)
```

Comment:
Changing `max_parallel_replicas` shouldn't affect this check. AFAIU, here we introduced delay in response for 4 replicas, so we expect that hedge connection mechanism will try to replace unresponsive replicas with another one (it checks `HedgedRequestsChangeReplica` profile event). Let's check with the author, @Avogar